src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferenceCache.cs

```diff
@@ -172,19 +164,18 @@ static bool IsMatch(ISyntaxFactsService syntaxFacts, string identifier, SyntaxTo
             return result.ToImmutableAndClear();
         }
 
-        static ImmutableArray<SyntaxToken> FindMatchingIdentifierTokensFromText(
-            ISyntaxFactsService syntaxFacts, string identifier, SyntaxNode root, SourceText sourceText, CancellationToken cancellationToken)
+        ImmutableArray<SyntaxToken> FindMatchingIdentifierTokensFromText()
```

Comment:
nit: I prefer these methods as static, a bit more verbosity is worth the clarity for me.

Reply:
made into normal methods to indicate they're not capturing anyting (beyond 'this' of course :)